config-example.yaml

```diff
@@ -142,11 +142,17 @@ db_path: /var/lib/headscale/db.sqlite
 # db_name: headscale
 # db_user: foo
 # db_pass: bar
+# db_max_idle_conns: 5
+# db_max_open_conns: 5
+# db_conn_max_idle_time_secs: 3600
```

Comment:
I think we need to prefix this with postgres, like `db_postgres_` or something as they do not apply to SQLite.

I might try to sneak a PR in before this to restructure the db config params because currently it is a bit chaotic, something like:

```yaml
db:
	type: postgres
	name: headscale
	user: foo
	password: bar
	postgres:
		max_idle_conns: 5
		max_open_conns: 5
```

or something like:

```yaml
db:
	type: postgres
	url: sql-style-url	

	sqlite:
		path: /var/run/db.sqlite

	# OR
	
	postgres:
		name: headscale
		user: foo
		password: bar
		max_idle_conns: 5
		max_open_conns: 5
```

I think I like the second option, and it can be implemented backwards compatible for a couple of versions with a warning of the change. What do you think? Makes it easier to implement dbURL style configs and sqlite specific stuff.

Reply:
Sorry for the late reply, I will see if I have time to make the change today/tomorrow and then you can rebase against it.

Reply:
Thank you.

Reply:
https://github.com/juanfont/headscale/pull/1700 is up now, please take a look.